From: Randy Dunlap <rdunlap@infradead.org>
Date: Sun, 13 May 2018 17:10:52 -0700
Subject: [PATCH] scsi: core: clean up generated file scsi_devinfo_tbl.c
References: bsc#1077989
Git-commit: a406b0a0693eafc6f6b3a633d25749370bf40d8c
Patch-mainline: v4.17-rc7

"make clean" should remove the generated file "scsi_devinfo_tbl.c", so
list it in the clean-files variable so that the file gets cleaned up.

Fixes: 345e29608b4b ("scsi: scsi: Export blacklist flags to sysfs")
Cc: Hannes Reinecke <hare@suse.de>
Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Reviewed-by: Hannes Reinecke <hare@suse.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/scsi/Makefile b/drivers/scsi/Makefile
index c4298c7fe819..6654ba1d462a 100644
--- a/drivers/scsi/Makefile
+++ b/drivers/scsi/Makefile
@@ -187,7 +187,7 @@ NCR_Q720_mod-objs	:= NCR_Q720.o ncr53c8xx.o
 oktagon_esp_mod-objs	:= oktagon_esp.o oktagon_io.o
 
 # Files generated that shall be removed upon make clean
-clean-files :=	53c700_d.h 53c700_u.h
+clean-files :=	53c700_d.h 53c700_u.h scsi_devinfo_tbl.c
 
 $(obj)/53c700.o $(MODVERDIR)/$(obj)/53c700.ver: $(obj)/53c700_d.h
 
-- 
2.12.3

